static/app/views/eventsV2/table/quickContext.tsx

```diff
@@ -33,49 +40,138 @@ export function hasContext(
   return isIssueContext(dataRow, column);
 }
 
+// NOTE: Will extend when we add more type of contexts.
+function getUrl(dataRow: TableDataRow, column: TableColumn<keyof TableDataRow>): string {
+  return isIssueContext(dataRow, column) ? `/issues/${dataRow['issue.id']}/` : '';
+}
+
+function fetchData(
+  api: Client,
+  dataRow: TableDataRow,
+  column: TableColumn<keyof TableDataRow>
+): Promise<IssueData> {
+  const promise: Promise<IssueData> = api.requestPromise(getUrl(dataRow, column), {
+    method: 'GET',
+    query: {
+      collapse: 'release',
+      expand: 'inbox',
+    },
+  });
```

Comment:
You don't have to tackle it in this PR, but I feel like the URL and query params should be returned from some function together. If you need to extend this for other URLs then presumably the query params are unique to that request as well

Reply:
Yeah, I have this in mind! Will address as we expand.